lightning_client.go

```diff
@@ -2703,3 +2708,57 @@ func (s *lightningClient) NetworkInfo(ctx context.Context) (*NetworkInfo,
 		NumZombieChans:       resp.NumZombieChans,
 	}, nil
 }
+
+// InvoiceSubscription holds the parameters to specify from which update
+// to start streaming.
+type InvoiceSubscription struct {
+	//If specified (non-zero), then we'll first start by sending out
+	//notifications for all added indexes with an add_index greater than this
+	//value. This allows callers to catch up on any events they missed while they
+	//weren't connected to the streaming RPC.
+	AddIndex uint64
+	//If specified (non-zero), then we'll first start by sending out
+	//notifications for all settled indexes with an settle_index greater than
+	//this value. This allows callers to catch up on any events they missed while
+	//they weren't connected to the streaming RPC.
+	SettleIndex uint64
+}
+
+// SubscribeInvoices allows a client to subscribe to updates
+// of newly added/settled invoices.
```

Comment:
nit: can use the full 80 char width for the comment.

Reply:
I've re-written the comment in order to reduce the length of it and fit it in one line